src/pages/landingpage/landing-page.js

```diff
@@ -1,7 +1,11 @@
 import localFont from "next/font/local";
-import { useState } from "react";
+import { useState,useEffect } from "react";
 import WhatCanICookAPI from '../api/WhatCanICookAPI.js';
+<<<<<<< HEAD
 import RecipesList from '../recipes/list.js';
+=======
+import Cookies from 'js-cookie';
+>>>>>>> d490ebd (Initialize cookie storing)
```

Comment:
Hi @mcdylanb, can you fix this conflict issue being highlighted above.

Also, while committing the fixes, can you use this command `git commit -S -m "YOUR_COMMIT_MESSAGE"` - this is to put signed commit needed before merging.

Thank you.